server/src/enum.ts

```diff
@@ -182,6 +182,12 @@ export enum UserStatus {
   DELETED = 'deleted',
 }
 
+export enum AssetStatus {
+  ACTIVE = 'active',
+  TRASHED = 'trashed',
+  DELETED = 'deleted',
+}
+
```

Comment:
I'm confused. Why do we need this?

Reply:
To immediately remove assets from the trash can while they are being deleted in the background.

Reply:
Meh, I wouldn't have added a status column just for that. You could have had a websocket event for "deletion in progress" with a spinner or similar.